servicetalk-http-api/src/test/java/io/servicetalk/http/api/DefaultHttpSetCookiesTest.java

```diff
@@ -769,6 +769,32 @@ private static void valueIteratorThrowsIfNoNextCall(HttpHeaders headers) {
         itr.remove();
     }
 
+    @Test
+    void throwIfNoSpaceBeforeCookieAttributeValue() {
+        final HttpHeaders headers = DefaultHttpHeadersFactory.INSTANCE.newHeaders();
+        headers.add("set-cookie", "first=12345;Extension");
+        headers.add("set-cookie", "second=12345;Expires=Mon, 22 Aug 2022 20:12:35 GMT");
+        throwIfNoSpaceBeforeCookieAttributeValue(headers);
```

Comment:
Can you please add tests to verify "lenient" mode when validation is disabled?
`DefaultHttpHeadersFactory` has ctor args to control validation.

Reply:
I added a test.